### DISCUSSION TITLE
[Feature] Allow flights to be associated with more than 1 user

### DISCUSSION THREAD

**Discussion by tomatojuice12** [johanohly/AirTrail#8] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Two ideas for implementation:

1. A multi-select dropdown of all users when editing/creating a flight. The world map shows you all flights for which you are tagged or that you own.

2. Under seat information, have the option to add 1 or more seats. The first is your own seat assigned automatically to your user, and any additional seats added can be assigned to an existing user, or freeform name input for non-members. The world map shows you all flights for which you have an assigned seat (even if seat number is NULL as this won't always be known for historic or future flights), or any flights you own.

For my use case (and probably for simpler implementation) it would be good for flights, and particularly notes, to be editable by anyone tagged on that flight. I don't imagine that creating the owner-selectable option of ReadOnly or ReadWrite sharing is worth the effort.

**Comment by johanohly** [comment]
I actually love your second idea, and I believe it is the better solution although it is a bit more complicated. 
For the changes in the database schema required to accomodate this, I will assign "seat"s to users instead of flights. The seat table would include seat information as well as a link to the flight.

**Comment by johanohly** [comment]
This had been added in v0.1.0!
